doc/api/process.markdown

```diff
@@ -16,6 +16,12 @@ continue.
 as [`process.exit()`][] or uncaught exceptions, and should not be used as an
 alternative to the `'exit'` event unless the intention is to schedule more work.
 
+## Event: 'disconnect'
+
+If Node.js was spawned with an IPC channel, this event will be emitted when
+either [`process.disconnect()`][] in child process or
+[`ChildProcess.disconnect()`] in parent process is called.
```

Comment:
This is mixing possible ways to cause disconnection with the actual behaviour.

disconnect will be emitted when the IPC channel is closed.

the IPC channel can be closed for many reasons: the ones mentioned above (explicit disconnect), but also, for example, when the parent process terminates.